tests/report_tests/plugin_tests/authd/entra_broker.conf

```diff
@@ -0,0 +1,4 @@
+[oidc]
+issuer = https://login.microsoftonline.com/a24144fa-4c27-4094-b21a-77c8fc56554a/v2.0
+client_id = 82e3e7f7-a1a3-4483-b782-9db24b8edc71
+force_provider_authentication = false
```

Comment:
Those are fake credentials, right?

I would like to double-confirm it before I merge it.

Same applies to google_broker.conf .

Reply:
Yes, I generated all the binary bits locally based on the shape of some real ones.